CMakeLists.txt

```diff
@@ -1,56 +1,47 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.1.3)
 
 include(cmake/base.cmake)
 include(cmake/boost.cmake)
-include(cmake/eigen.cmake)
+include(cmake/msvc-specific.cmake)
 
 set(PROJECT_NAME eigen-quadprog)
 set(PROJECT_DESCRIPTION "QuadProg QP solver through Eigen3 library.")
 set(PROJECT_URL "https://github.com/jorisv/RBDyn/eigen-quadprog")
-
-project(${PROJECT_NAME} CXX)
+set(PROJECT_DEBUG_POSTFIX "_d")
 
 # Disable -Werror on Unix for now.
 set(CXX_DISABLE_WERROR True)
 set(DOXYGEN_USE_MATHJAX "YES")
 
 option(USE_F2C "Use f2c converted code (usually slower)." OFF)
-option(USE_FORTRAN_SUBDIRECTORY "Use cmake to generate sub project for fortran parts" OFF)
-
 if(WIN32)
-  set(USE_F2C OFF)
-  set(USE_FORTRAN_SUBDIRECTORY ON)
+  set(USE_FORTRAN_SUBDIRECTORY_DEFAULT ON)
+else()
+  set(USE_FORTRAN_SUBDIRECTORY_DEFAULT OFF)
 endif()
+option(USE_FORTRAN_SUBDIRECTORY "Use cmake to generate sub project for fortran parts" ${USE_FORTRAN_SUBDIRECTORY_DEFAULT})
+option(DISABLE_TESTS "Disable unit tests." OFF)
 
 setup_project()
+set(PROJECT_VERSION 1.0.0)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
-
-if(UNIX)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic")
-endif()
+add_project_dependency(Eigen3 REQUIRED)
 
-#########################
-# External dependencies
-if(PKG_CONFIG_FOUND)
-  search_for_eigen()
-else()
-  find_path(Eigen_INCLUDE_DIR NAMES signature_of_eigen3_matrix_library
-    PATHS
-    ${CMAKE_INSTALL_PREFIX}/include
-    PATH_SUFFIXES eigen3 eigen
-  )
-  include_directories(SYSTEM ${Eigen_INCLUDE_DIR})
-endif()
+if(MSVC)
+  if(USE_FORTRAN_SUBDIRECTORY)
+    set(CMAKE_MSVCIDE_RUN_PATH "\$(SolutionDir)/src/\$(Configuration);\$(SolutionDir)/src/qld_fortran")
```

Comment:
That's not the right path. Should be `quadprog_fortran` instead of `qld_fortran`

Reply:
Right.
Fixed in next commit.